src/skywallet/transaction.go

```diff
@@ -86,7 +105,7 @@ func (s *SkycoinTransactionSigner) Sign() ([]string, error) {
 					}
 					index += 8
 				} else {
-					return nil, fmt.Errorf("protocol error: expected TXINPUT, but got %s", *txRequest.RequestType)
+					return nil, ErrUnexpectedTxinput
```

Comment:
dont think you need to make these error constants as the error is variable 
makes sense  for errors like `empty inputs` but here its better to use 
`fmt.Errorf("protocol error: expected TXINPUT, but got %s", *txRequest.RequestType)`

Reply:
It was my bad, as it is switch depending on txRequest.RequestType above, so here will be always the same txRequest.RequestType - TXINPUT, so this error actually is not variable.